<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PMIG</class>
 <widget class="QMainWindow" name="PMIG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PMIG</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayoutRight">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="functional">
         <attribute name="title">
          <string>Functional</string>
         </attribute>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QListWidget" name="listWidget_Functional">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="convolutional">
         <attribute name="title">
          <string>Convolutional</string>
         </attribute>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QListWidget" name="listWidget_Convolutional"/>
          </item>
          <item row="1" column="0">
           <widget class="QFrame" name="frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="3" column="0">
              <widget class="QGroupBox" name="groupBox_2">
               <layout class="QGridLayout" name="gridLayout_5">
                <property name="topMargin">
                 <number>9</number>
                </property>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="lineEdit"/>
                </item>
                <item row="1" column="2">
                 <widget class="QPushButton" name="pushButton">
                  <property name="text">
                   <string>Save</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QFrame" name="frame">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="2" column="1">
                 <widget class="QSpinBox" name="spinBox_3"/>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Divisor</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Kernel Height</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Offset</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Kernel Width</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="spinBox_2"/>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="spinBox"/>
                </item>
                <item row="5" column="0">
                 <widget class="QPushButton" name="pushButton_3">
                  <property name="text">
                   <string>Calculate Divisor</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QPushButton" name="pushButton_2">
                  <property name="text">
                   <string>Apply Size</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="CustomFilterWidget::CustomConvFilterTable" name="customConvTable"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="Widgets::ImageView" name="graphicsViewLeft"/>
      </item>
      <item row="0" column="1">
       <widget class="Widgets::ImageView" name="graphicsViewRight"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionReset_Image"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionReset_Image">
   <property name="text">
    <string>Reset Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widgets::ImageView</class>
   <extends>QGraphicsView</extends>
   <header>imageview.h</header>
  </customwidget>
  <customwidget>
   <class>CustomFilterWidget::CustomConvFilterTable</class>
   <extends>QTableWidget</extends>
   <header>customconvfiltertable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
